custom/icds/tasks/sms.py

```diff
@@ -118,3 +124,35 @@ def send_sms_limit_exceeded_alert(sender, instance, **kwargs):
             """).format(env=settings.SERVER_ENVIRONMENT, date=instance.date, sms_limit=sms_daily_limit)
         send_html_email_async.delay(subject, recipients, message,
                 email_from=settings.DEFAULT_FROM_EMAIL)
+
+
+@task
+def delete_sms_events(start_date, end_date):
```

Comment:
Should we add a validation or a warning on `end_date` to not delete very recent ones?